python/psautohint/otfFont.py

```diff
@@ -1068,45 +1068,13 @@ def updateFromBez(self, bezData, glyphName, width):
                         glyphName)
 
     def saveChanges(self):
-        ttFont = self.ttFont
-        font_format = self.font_format
-        inputPath = self.inputPath
-        outFilePath = self.outFilePath
-
-        overwriteOriginal = False
-        if inputPath == outFilePath:
```

Comment:
Can this check be really removed?

Reply:
Why not? We no longer care whether we are saving to a new file or overwriting existing one.

Reply:
From the POV of a user we care, that's why I asked. But you say below that __init__ takes care of it